dvf_json/src/Plugin/Visualisation/Source/JsonFile.php

```diff
@@ -231,6 +232,10 @@ protected function fetchData() {
     $data = '{}';
 
     if ($response) {
+      $dvfHelpers = new DvfHelpers();
```

Comment:
@WilsonWu2100 I believe it would be better to move this to a property, instantiated within the JsonFile constructor, e.g:
`$this->dvfHelpers = new DvfHelpers();`

This way if we need to use it elsewhere within the class we can just do:
`$this->dvfHelpers->validateJson()`

Reply:
Fixed